src/app/youtube/pipes/search.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { SearchItem } from 'src/app/youtube/models/search-item.model';
+
+@Pipe({
+    name: 'searchName'
+})
+export class SearchPipe implements PipeTransform {
+    public transform(searchResults: SearchItem[], inputValue: string): SearchItem[] {
+        return searchResults
+            .filter(searchItem =>
+                searchItem.snippet.title.toUpperCase().toLowerCase()
```

Comment:
What do you need `toUpperCase()` here for? It seems just `toLowerCase()` will be enough